cmd/nerdctl/event/events.go

```diff
@@ -26,6 +26,8 @@ import (
 
 	"github.com/containerd/containerd/events"
 	"github.com/containerd/containerd/log"
+	client2 "github.com/containerd/nerdctl/cmd/nerdctl/client"
```

Comment:
I'd prefer the package name to be like `ncclient`

Reply:
Yes, I will clean dirty name when I pass all CI(lol

Reply:
another suggestion is `ndclient` 😃  since `nd` reads more similar to `nerd`ctl

Reply:
I'd argue that the contaiNERD client library is also "nerd" 🤓